### DISCUSSION TITLE
Allow achievements to trigger in all normal worlds

### DISCUSSION THREAD

**Discussion by KioProject123** [PurpurMC/Purpur#1164] [__Dismissed__] [answered]
There are some custom achievement data packets that can only be triggered in the Overworld. This can only be triggered in the default world and not in plugin-created worlds.
It would be nice if there was an option to allow all overworlds to trigger

**Comment by RhythmicSys** [comment, marked as answer]
This would be an issue with the datapack, if you're placing the datapack in the main world file, it should be affecting all worlds
